### ISSUE TITLE
NO dropout in MLP and CausalSelfAttention

### ISSUE THREAD

**Issue by peter-ni-noob** [karpathy/build-nanogpt#29]

**Comment by unclecode** [comment]
Yes, he elaborated on this topic in his video. Overfitting is not a major concern for this project overall because the dataset is virtually infinite. Even after 4 epochs and 40 billion tokens, the validation loss is decreasing steadily.

**Comment by peter-ni-noob** [comment]
ok
